Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12.13.1-alpine as development
```

Comment:
@Bazillius could you look?
Worth to have  the same approach as for API

Reply:
@melnychukvitaliy sounds good!